ceramic/schemas/db_stream_schema.json

```diff
@@ -15,6 +15,9 @@
                 },
                 "timestamp": {
                     "type": ["integer", "null"]
+                },
+                "counted_for_activity": {
+                    "type": ["boolean"]
```

Comment:
Updated the data model to include whether tweets have been already counted for rewards